lib/plugins/Label_H1_POS.ts

```diff
@@ -88,7 +87,42 @@ export class Label_H1_POS extends DecoderPlugin {
 
       decodeResult.decoded = true;
       decodeResult.decoder.decodeLevel = 'partial';
-    } else if(parts.length === 32) { // #M1B long variant
+    } else if(parts.length === 15) { // variant 6
+    decodeResult.remaining.text = ''
+    processUnknown(decodeResult, parts[0]);
+    processUnknown(decodeResult, parts[1]);
+    let date = undefined;
+    if(parts[2].startsWith('/DC')) {
+      date = parts[2].substring(3);
+    } else {
+      processUnknown(decodeResult, parts[2]);
+    }
+    processUnknown(decodeResult, parts[3]);
+    const fields = parts[4].split('/');
+    for(let i=0; i<fields.length; ++i) {
+      const field = fields[i];
+      if(field.startsWith('PS')) {
+        processPosition(decodeResult, field.substring(2));
+      } else {
+        if(i===0) {
+          processUnknown(decodeResult, field);
+        } else {
+          processUnknown(decodeResult, '/'+field);
+        }
+      }
+    }
+    processRoute(decodeResult, parts[7], parts[5], parts[9], parts[8], undefined, date);
```

Comment:
the timestamps might not be correct here. they definitely look like timestamps but are very far apart